src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -58,10 +48,23 @@ public static function get_all() {
 				'image'      => WC()->plugin_url() . '/assets/images/stripe.png',
 				'plugins'    => array( 'woocommerce-gateway-stripe' ),
 				'is_visible' => array(
+					self::get_rules_for_countries( OnboardingTasks::get_stripe_supported_countries() ),
 					(object) array(
-						'type'     => 'or',
-						'operands' => $stripe_countries_rules,
+						'type'        => 'option',
+						'option_name' => 'woocommerce_onboarding_profile',
+						'value'       => 'cbd-other-hemp-derived-products',
+						'operation'   => '!contains',
```

Comment:
I've seen this check in a few places and it doesn't work. The reason is because industry isn't a simple array

<img width="462" alt="Screen Shot 2021-05-25 at 11 50 03 AM" src="https://user-images.githubusercontent.com/1922453/119419980-794baa00-bd4f-11eb-8d27-46f331636929.png">

This is why transformers were added in https://github.com/woocommerce/woocommerce-admin/pull/6948. See the readme of that PR for the correct implementation. I know several Remote Inbox Notifications are guilty of this as well.

Reply:
Gotcha.  Updated this in https://github.com/woocommerce/woocommerce-admin/pull/7025/commits/f42004e6e2d531f93c1f1cd403e6f679c4e6d5fc